staging/src/k8s.io/apiserver/pkg/server/genericapiserver.go

```diff
@@ -439,12 +428,23 @@ func (s *GenericAPIServer) PrepareRun() preparedGenericAPIServer {
 //           |                                                       |
 //  AfterShutdownDelayDuration (delayedStopCh)   PreShutdownHooksStopped (preShutdownHooksHasStoppedCh)
```

Comment:
```suggestion
//                                                                               PreShutdownHooksStopped (preShutdownHooksHasStoppedCh)
```
`AfterShutdownDelayDuration` was removed , right?

Reply:
oh, no, nvm , is not removed, we add anew `NotAcceptingNewRequest` lifecycle signal